oedi/config.yaml

```diff
@@ -26,6 +59,7 @@ AWS:
         - s3://oedi-data-lake/tracking-the-sun/2021/
         - s3://oedi-data-lake/tracking-the-sun/2022/
         - s3://oedi-data-lake/tracking-the-sun/2023/
+      Table Prefix: tracking_the_sun_
```

Comment:
I am assuming the Table Prefix matches Locations using `list` index, right? Add multiple `tracking_the_sun_` prefixes and match the number of locations?

Keep the structure same between all dataset, so that no confuse people.

Reply:
Keeping the option for Table Prefix in addition to Table Prefixes, as discussed.